tests/testsuite.src/run_fundamental.at

```diff
@@ -7203,6 +7221,11 @@ FROM SIGN TRAILING SEPARATE
  ===>+     12,34<===
  ===>-     12,34<===
  ===>-     12,34<===
+EXTRA FROM SIGN LEADING SEPARATE
+ ===>           <===
+ ===>           <===
```

Comment:
I _think_ that should be fine, showing "1" if before `switch (c) {` (around line 1123) you add something like `src_num = COB_D2I(*src)` and use this instead of *src for every part below that does not check for something that's not expected to be not numeric.

Reply:
Was I wrong about this - as there still is no "1" for the "A"?

Reply:
Oh, I was missing two MOVEs in the last test ; now I get the "A"s.

I'd have to check on GCOS (can't do it now, on a business trip), but AFAIR, it does not perform "normalization" when moving to NUMERIC-EDITED (unlike MF, apparently).

Reply:
Okay, works as expected on GCOS.

However, this patch breaks one test, namely "Figurative constants to numeric field" in `run_misc.at:7431`:
```quote
-NUM9 value SPACES is       .
+NUM9 value SPACES is 000000.
```